website/source/api/auth/okta/index.html.md

```diff
@@ -264,12 +268,14 @@ Registers a new group and maps a set of policies to it.
 ### Parameters
 
 - `name` `(string: <required>)` - The name of the group.
-- `policies` `(string: "")` - Comma-separated list of policies associated with
-  the group.
+- `policies` `(policies: [])` - The list of policies associated with the group.
```

Comment:
`policies` isn't a JSON type you also got rid of the information that the input can be comma-separated. Please indicate that it can be a comma-separated string or a JSON array of strings.

Reply:
fixed